test/e2e/main_test.go

```diff
@@ -0,0 +1,44 @@
+package e2e
+
+import (
+	"testing"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	apis "github.com/nmstate/kubernetes-nmstate/pkg/apis"
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+)
+
+var (
+	f         = framework.Global
+	t         *testing.T
+	namespace string
+	nodes     = []string{"node01"} // TODO: Get it from cluster
+
+	_ = BeforeSuite(func() {
```

Comment:
no need to keep those under var, you can put them to the root

Reply:
Done